### DISCUSSION TITLE
Does MRTK3 support eye tracking with meta quest pro?

### DISCUSSION THREAD

**Discussion by jockeyc** [MixedRealityToolkit/MixedRealityToolkit-Unity#623] [Q&A]
Hand interaction works fine in my project, but eye tracking doesn't work.
The documentation for the xr interaction toolkit has samples using the meta gaze adaptor. And has the following tips.

> Currently the XRGazeInteractor uses the [OpenXR bindings] for eye gaze position, tracking, and rotation. This binding is not currently supported for Quest Pro. When the OpenXR bindings are supported, this sample and adapter will be unnecessary.

I was able to get eye movement data when I checked Oculus in XR Plug-in Management. But MRTK3 needs to use OpenXR. but in XR Plug-in Management the two are in conflict with each other. Am I looking in the wrong direction to use eye movement on quest pro.

**Comment by AMollis** [comment]
@keveleigh / @david-c-kline   ... do you know the answer to this question?
